### PR TITLE
add upboard rosinstall

### PR DESCRIPTION
**Pull request by knorth55**
add upboard rosinstall for baxter system

### CODE REVIEW

File: jsk_arc2017_common/data/models/fcn32s.npz [hunk 1]
```diff
@@ -0,0 +1 @@
+/home/baxter/ros/kinetic/src/start-jsk/jsk_apc/jsk_arc2017_common/data/models/fcn32s_arc2017_datasetv3_cfg009_20170724.npz
```
Comment (pazeshun) [inline jsk_arc2017_common/data/models/fcn32s.npz]: Please remove this file.

Comment (knorth55) [reply jsk_arc2017_common/data/models/fcn32s.npz]: updated

**Comment by knorth55** [comment]
RethinkRobotics/rethink_ee_description
do we really need this?

**Comment by pazeshun** [comment]
> RethinkRobotics/rethink_ee_description
> do we really need this?

Not really in `catkin build`.
But without this, `rosdep` outputs error because `baxter_description` depends on `rethink_ee_description`

**Comment by pazeshun** [comment]
https://github.com/start-jsk/jsk_apc/pull/2678/commits/1863493c074bc106bf60684f53bb91f0a45a4930: Generating `baxter.l` in `catkin build` failed due to lack of `baxter_description`.
https://github.com/start-jsk/jsk_apc/pull/2678/commits/6a962d3469f748401ffd6d866106c9c552903660, https://github.com/start-jsk/jsk_apc/pull/2678/commits/4bc45e25bf5db9540be977e6f9b4b51a46f3a7b0: Add README for UP Board installation.

**Comment by pazeshun** [comment]
Without https://github.com/start-jsk/jsk_apc/pull/2678/commits/599749e3f76d0dd0361d8f46135aad418f73281f, the following error comes from dynamixel node on baxter-c2:
```
OverflowError: unsigned byte integer is less than minimum
```

**Comment by knorth55** [comment]
can i merge this PR?

**Comment by pazeshun** [comment]
@knorth55 
Sorry for late.
I updated README and finished this PR.
Please check this.

**Comment by knorth55** [comment]
LGTM!

**Pull request merged**